src/components/TvDetails/index.tsx

```
@@ -461,9 +466,14 @@ const TvDetails = ({ tv }: TvDetailsProps) => {
     type: 'or',
   });
 
+  const { theme } = useTheme();
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)